en/concepts/scene/scene-editing.md

```diff
@@ -0,0 +1,95 @@
+The following shortcuts work for both views.
+
+- Select a node in the **Hierarchy** panel and double-click or press <kbd>Cmd/Ctrl + F</kbd> to focus the node in the **Scene** panel.
+- Pressing <kbd>Cmd/Ctrl + D</kbd> after selecting a node will copy and paste an identical node at the same location as the node, which can be used for efficiency when we need to make multiple similar nodes quickly.
+- In **Scene**/**Hierarchy** panel, hold down the <kbd>Cmd/Ctrl</kbd> key and click on the node you want in order to select multiple nodes at the same time. The <kbd>Shift</kbd> key allows you to select nodes in succession, without selecting them one by one.
+
+### Align/Evenly Distribute Nodes
+
+When the **Scene** panel is in **2D** view, there is a row of buttons in the upper left corner that can be used to align or evenly distribute multiple nodes when they are selected. The specific rules are as follows:
+
+![alignment](scene-editing/alignment.png)
+
+Assuming that all three Label nodes are selected, the six alignment buttons from left to right will align these nodes in turn:
+
+- top-aligned to the nearest upper border (not the upper border of the topmost node)
+- vertically centered, aligned to the overall horizontal center line
+- bottom-aligned, aligned to the nearest lower boundary
+- Left-aligned to its closest left border
+- Horizontally centered, aligned to the overall vertical center line
+- Right-aligned, aligned to the nearest right border
+
+The 6 distribution buttons in the second half, from left to right, will align these nodes in turn:
+
+- Top distribution, evenly distributed along the top boundary of the node
+- Vertically centered distribution, evenly distributed along the horizontal midline of the node
+- Bottom distribution, evenly distributed along the lower boundary of the node
+- Left distribution, evenly distributed along the left border of the node
+- Horizontally centered distribution, evenly distributed along the vertical center line of the node
+- Right distribution, evenly distributed along the right boundary of the node
+
+> **Note**: whether the left and right boundaries and the center line are determined at the beginning, or the reference when aligning/evenly distributing each node later, it is the center of the node bounding box or one of the boundaries rather than the position coordinates of the nodes.
+
+For example, in the figure below, when we align the three Label nodes with different widths to the right, we get the alignment of the right boundaries of the three node bounding boxes, not the `x` coordinates of the three node positions becoming the same.
+
+![align to right](scene-editing/align-to-right.png)
+
+## Scene Display Effects
+
+There is also support for setting skyboxes, global fog effects, and shadows in the scene to better enrich the scene and render and display the scene environment. For details, please refer to:
```

Comment:
1. ......for setting the Skybox, Global Fog and Shadow？
2. ......to better enrich the scene, render and display the scene environment.